docs/astrocut/index.rst

```diff
@@ -219,7 +219,7 @@ Making cutout target pixel files
 --------------------------------
 
 To make a cutout, you must already have an image cube to cut out from.
-Assuming that that step has been completed, you simply give the central
+Assuming that you have a TESS cube file stored locally, you can simply give the central
```

Comment:
I know not your change, but can we remove all instances of the word "simply" from our docs?